src/kdbindings/node_functions.h

```diff
@@ -55,7 +55,7 @@ struct any_bindables<HEAD, Ts...> : std::integral_constant<
     template<typename... Ts>                                                                                                                                                                          \
     inline auto NAME(Ts &&...args)->std::enable_if_t<KDBindings::Private::any_bindables<Ts...>::value, KDBindings::Private::Node<KDBindings::Private::operator_node_result_t<decltype(FUNC), Ts...>>> \
     {                                                                                                                                                                                                 \
-        return Private::makeNode(FUNC, std::forward<Ts>(args)...);                                                                                                                                    \
+        return KDBindings::Private::makeNode(FUNC, std::forward<Ts>(args)...); \
```

Comment:
```suggestion
        return ::KDBindings::Private::makeNode(FUNC, std::forward<Ts>(args)...); \
```

Good catch, to make sure this works from any namespace, we should make this an absolute path.

Reply:
Done